app/src/main/kotlin/com/hippo/ehviewer/ui/MainActivity.kt

```diff
@@ -530,16 +530,16 @@ class MainActivity : EhActivity() {
 
 val LocalNavDrawerState = compositionLocalOf<DrawerState> { error("CompositionLocal LocalNavDrawerState not present!") }
 val LocalSideSheetState = compositionLocalOf<DrawerState2> { error("CompositionLocal LocalSideSheetState not present!") }
-val LocalDrawerLockHandle = compositionLocalOf<SnapshotStateList<Int>> { error("CompositionLocal LocalDrawerLockHandle not present!") }
+val LocalDrawerHandle = compositionLocalOf<SnapshotStateList<Int>> { error("CompositionLocal LocalDrawerHandle not present!") }
 val LocalSnackBarHostState = compositionLocalOf<SnackbarHostState> { error("CompositionLocal LocalSnackBarHostState not present!") }
 val LocalSnackBarFabPadding = compositionLocalOf<State<Dp>> { error("CompositionLocal LocalSnackBarFabPadding not present!") }
 
 @Composable
-fun LockDrawer(value: Boolean) {
-    val updated by rememberUpdatedState(value)
+fun DrawerHandle(enabled: Boolean) {
+    val updated by rememberUpdatedState(enabled)
```

Comment:
This `rememberUpdatedState` should be redundant.

Reply:
Done